File: python/free_scrub.py
import json
import urllib.request

body = json.dumps({"text": "Hello world", "mode": "standard"}).encode()
req = urllib.request.Request(
    "https://api.scrubmytext.com/v1/scrub",
    data=body,
    headers={"Content-Type": "application/json"},
    method="POST",
)
with urllib.request.urlopen(req) as r:
    print(r.read().decode())

File: javascript/free-scrub.mjs
const response = await fetch("https://api.scrubmytext.com/v1/scrub", {
  method: "POST",
  headers: { "Content-Type": "application/json" },
  body: JSON.stringify({ text: "Hello world", mode: "standard" }),
});
console.log(await response.json());
